### PR TITLE
dockerize dev enviroment

### PR DESCRIPTION
**Pull request by trddddd**

### CODE REVIEW

File: app.dockerfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
+FROM ruby:2.5
```
Comment (Mehonoshin) [inline app.dockerfile]: Let's call this file `Dockerfile.dev`

Comment (Mehonoshin) [reply app.dockerfile]: What is the purpose of having a separate dockerfile? What is the difference between this file and regular Dockerfile?

Comment (trddddd) [reply app.dockerfile]: Live reload code and preset chrome headless for capybara tests into docker.

Comment (Mehonoshin) [reply app.dockerfile]: Okay, then just rename it and add a screenshot please

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,59 +1,73 @@
...
+    DatabaseCleaner.strategy = if example.metadata[:disable_transaction]
+                                 :truncation
+                               else
+                                 :transaction
+                               end
     DatabaseCleaner.start
   end
 
   config.before(:each, type: :feature) do
     # Для капибарных тестов заполним редис курсами, чтобы не пытался стянуть их из инета
```
Comment (Mehonoshin) [inline spec/spec_helper.rb]: Also please rewrite such Russian comments in English. The project is international, and English-speaking developers may use it.

**Comment by trddddd** [comment]
> Please provide some screenshots how does it look on local env, for example when you run specs in docker

https://www.youtube.com/watch?v=VFRKPO5LHDg

File: .env.test [hunk 3]
```diff
@@ -1,3 +1,5 @@
+RAILS_ENV=test
```
Comment (Mehonoshin) [inline .env.test]: Это по идее не нужно, dotenv подтянет этот конфиг только когда RAILS_ENV уже установлен в test

**Pull request merged**